tools-and-communication/creating-videos.instructor.md

```diff
@@ -0,0 +1,76 @@
+    - Feel free to add the finished product to the Learn lesson as well!  🐥🐥 for 🪨
+- **Come Back to the learning goals**
+  - Talk about the learning objectives in the beginning.
+  - Come back to them at the end and connect them to the topics discussed (great review)
+- **Keep it short!**
+  - The ideal video should be ~10 minutes.  Try never to go more than ~20 as that becomes a real slog for students.  It's also harder for them to focus in on a difficult concept if they have to hunt through the video to find things.
+  - You can break up a long lesson into short videos and use a playlist to put 'em together.
+- **When you remember add captions**
+  - Panopto can do automatic captions, but you have to go into the video in the site and "import automatic captions".
+![Automatic Captions](../assets/automatic-captions.png)
+- **Make mistakes**
+  - I (Chris) am going to maintain I do this intentionally, and you can't prove otherwise.
+  - Mistakes in the video allow us to pause and ask the student, "What happened here?"  Give them a chance to guess at the answer and/or pause the video to try to solve the problem.  Then you can show them how you resolve the problem/mistake.
+  - You can work errors into your lessons and ask students to take a moment to solve the problem, and then make it into a mini-lesson about how to debug or research problems.
+  - Mistakes also help humanize you to students
+- **Give practice**
+  - Mistakes above can serve this purpose, but you can give students a question or problem and ask them to pause the video and figure it out, and then work through how you resolve things.
+  - You can also use the Learn Challenges in the curriculum.
+- **Be Aware of where the thumbnail will be**
+  - Both Panopto and Zoom put thumbnails of the speaker in a fixed location in the video.  *Don't put code where it will be covered up!*
+  - Camtasia does allow you to shift the location of the thumbnail in editing.
+
+## Workflow
+
+To help adding video lessons the flow goes like this:
+
+1.  Find the [Video content Airtable](https://airtable.com/tblXx1WLkPwJqp46T)
+1.  Identify a video assigned to you, or pick one which is unclaimed
+1.  Review the lesson, make suggestions/edits for the curriculum team.
+1.  Mark the video as "In Progress" in the airtable
+1.  Record the lesson making edits as needed
+1.  Upload the video lessons to [panopto](https://adaacademy.hosted.panopto.com/Panopto/Pages/Sessions/List.aspx#folderID=%22bf8fa6c1-e3d3-46b2-b3f9-accb01698ec1%22)
+1.  Create a playlist (even for a 1 video lesson) and attach videos to it.
+1.  Place the embedded iframe for the playlist into the learn lesson.
+1.  Git-add-commit and learn publish the lesson
+1.  Watch at least a bit of each video, just to make SURE you didn't make a mistake 😊
+1.  Mark the lesson as done in the airtable.
+
+## Questions To Answer Over Time
```

Comment:
I'm realizing I think it'd be nice to add another section here creating norms about Panopto folder/playlist/video organization. I'm happy to take a stab at that in a new PR after you push this one, if that's easiest!

Reply:
I pushed in my suggested org, and feel free to change it in a PR.